### ISSUE TITLE
Clicking breadcrumb to go to current folder doesn't work in column view

### ISSUE THREAD

**Issue by GammaGames** [elementary/files#1498]
## Describe the bug
Clicking the last breadcrumb in the path should (at least it *used to*) make the first column the target folder.

### To Reproduce
1. Set view to columns
2. Go to nested folder
3. Click on the last breadcrumb
4. Nothing happens

### Expected behavior
It should clear all columns and move the last column to the far left, like it does if you click any other part of the breadcrumb

### Screenshots or screen recordings
![Screen record from 2020-10-09 11 26 02](https://user-images.githubusercontent.com/7832163/95614922-9633a700-0a24-11eb-896e-cea59f00f597.gif)

## Platform Information
![image](https://user-images.githubusercontent.com/7832163/95615023-c5e2af00-0a24-11eb-8ae1-a4c61dff26f1.png)

- I'm using the latest released stable version

**Comment by jeremypw** [comment]
Doesn't seem any good reason why the last breadcrumb should differ in behaviour.  If it helps pending a fix, you can also double click on a folder in the view to make it root.

**Comment by GammaGames** [comment]
I thought it was an optimization for the non-column views. Instead of having to recreate the view for the folder the app is currently showing, it just ignored the action altogether. I only noticed because columns is the only view I use.

I have single click disabled with tweaks so I can't make make the folder root by double clicking. I don't mind that, I did it to myself with the non-standard setup.
